modules/javafx.web/src/main/java/com/sun/javafx/webkit/prism/WCFontImpl.java

```diff
@@ -160,6 +160,12 @@ private FontStrike getFontStrike()
         return getFontStrike().getFontResource().getAdvance(glyph, font.getSize());
     }
 
+    @Override public float[] getGlyphBoundingBox(int glyph) {
+        float[] bb = new float[4];
```

Comment:
You are right, this method seems often used, I think that WebCore displays the mathematics words glyph by glyph due the nature of these words (frequently 1 or 2 characters) and over all because some of theses glyphs doesn't respect ascent and descent. But I made no test.

My purpose was simply to repair MathML rendering with a minimum of changes, not improve performances or memory usage. I think you are far more qualified than me to appreciate that. So I'm rely on you.

Do you means that, that all variables not directly used by others method of the class or not directly called could be declare private and static.
float[] bb in getGlyphBoundingBox(int glyph), for example ?
In this case, it should be interresting to test all methods ?
May be you already did that ?

Or I can not declare this variable and code like that :
```
return  new float[]{
getFontStrike().getFontResource().getGlyphBoundingBox(glyph, font.getSize(), bb)[0],
-getFontStrike().getFontResource().getGlyphBoundingBox(glyph, font.getSize())[3], 
getFontStrike().getFontResource().getGlyphBoundingBox(glyph, font.getSize())[2], 
getFontStrike().getFontResource().getGlyphBoundingBox(glyph, font.getSize())[3]
- getFontStrike().getFontResource().getGlyphBoundingBox(glyph, font.getSize())[1]};
```

Reply:
Ok.